update Transifex instructions (#12156)
openstreetmap/iD: CONTRIBUTING.md

[Transifex](https://app.transifex.com/openstreetmap/id-editor/) platform. Sign up to Transifex via the
["Join This Project"](https://app.transifex.com/join/?o=openstreetmap&p=id-editor&t=opensource) link on
the [iD's project page](https://app.transifex.com/openstreetmap/id-editor/) and
click **Translate** to start translating. The registration form asks for a "business email" but you may provide
a regular one. If you try to join via the Transifex home page "Sign up" link you may be unable to create an

> account. After registration your account must be manually approved by the project's maintainers; you won't be able to translate until that happens.

Author: ```suggestion
account. After registration your account must be approved; you won't be able to translate until that happens.
```

[Transifex](https://app.transifex.com/openstreetmap/id-editor/) platform. Sign up to Transifex via the
["Join This Project"](https://app.transifex.com/join/?o=openstreetmap&p=id-editor&t=opensource) link on
the [iD's project page](https://app.transifex.com/openstreetmap/id-editor/) and
click **Translate** to start translating. The registration form asks for a "business email" but you may provide
a regular one. If you try to join via the Transifex home page "Sign up" link you may be unable to create an

> account. After registration your account must be manually approved, you are unable to translate until that happens.

Reviewer: ```suggestion
account. After registration your account must be manually approved by the project's maintainers, you are unable to translate until that happens.
```

> account. After registration your account must be manually approved by the project's maintainers, you are unable to translate until that happens.

Reviewer 2: ```suggestion
account. After registration your account must be manually approved by the project's maintainers; you won't be able to translate until that happens.
```

changed to sound little better (to me)
